Utilities/EasyMarkup/EmPropertyT.cs

```diff
@@ -75,9 +75,9 @@ public virtual T ConvertFromSerial(string value)
         internal override EmProperty Copy()
```

Comment:
It may be possible to handle the `Copy()` method at the top level base classes completely by using `GetType()` to get the runtime class of the instance.  
https://docs.microsoft.com/en-us/dotnet/api/system.object.gettype?view=netframework-4.7.2  
As long as the constructors are as they need to be, this could free up a lot of unneeded overrides.

Reply:
Will have to be done with the EaryMarkup overhaul. Not now.